includes/wp-admin/class-settings.php

```diff
@@ -442,6 +443,20 @@ public static function add_settings_help_tab() {
 		);
 	}
 
+	/**
+	 * Handle 'welcome' query arg.
+	 */
+	public static function handle_welcome_query_arg() {
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		if ( isset( $_GET['welcome'] ) ) {
+			// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			$welcome_checked = empty( \sanitize_text_field( \wp_unslash( $_GET['welcome'] ) ) ) ? 0 : 1;
+			\update_user_meta( \get_current_user_id(), 'activitypub_show_welcome_tab', $welcome_checked );
+			\wp_safe_redirect( \admin_url( 'admin.php?page=activitypub&tab=settings' ) );
```

Comment:
```suggestion
			\wp_safe_redirect( \admin_url( 'options-general.php?page=activitypub&tab=settings' ) );
```
We use `options-general.php` in other places

Reply:
AI 🫣